### PR TITLE
CSP-1088 Refactored member profile view in partials

### PR DESCRIPTION
**Pull request by DevenKShah**

### CODE REVIEW

File: src/SFA.DAS.Aan.SharedUi/Models/PublicProfile/ConnectWithMemberSubmitModelValidator.cs [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+using SFA.DAS.Aan.SharedUi.Models.PublicProfile;
 
 namespace SFA.DAS.ApprenticeAan.Web.Validators.MemberProfile;
 
-public class SubmitConnectionCommandValidator : AbstractValidator<SubmitConnectionCommand>
+public class ConnectWithMemberSubmitModelValidator : AbstractValidator<ConnectWithMemberSubmitModel>
 {
     public const string ReasonToConnectValidationMessage = "You must tell us why you want to get in touch";
     public const string HasAgreedToSharePersonalDetailsValidationMessage = "You must agree to sharing your details";
     public const string HasAgreedToCodeOfConductValidationMessage = "You must agree to adhere the code of conduct";
```
Comment (MarkFCain) [inline src/SFA.DAS.Aan.SharedUi/Models/PublicProfile/ConnectWithMemberSubmitModelValidator.cs:10]: this looks like a typo, probably 'you must agree to adhere to the code of conduct'

I know this isn't new code/words

File: src/SFA.DAS.Aan.SharedUi/Services/MemberProfileHelper.cs [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    public static string? GetProfileValueByDescription(string profileDescription, List<Profile> profiles, IEnumerable<MemberProfile> memberProfiles)
+    {
+        var profileId = profiles.SingleOrDefault(p => p.Description.Equals(profileDescription, StringComparison.OrdinalIgnoreCase))?.Id;
+        if (profileId == null) return null;
+        return memberProfiles.FirstOrDefault(m => m.ProfileId == profileId)?.Value;
+    }
+
+    public static string? GetEmployerAddress(IEnumerable<MemberProfile> memberProfiles)
+    {
+        var employerAddressProfileIds = new int[] { 31, 32, 33, 34, 35 };
```
Comment (MarkFCain) [inline src/SFA.DAS.Aan.SharedUi/Services/MemberProfileHelper.cs:30]: can we convert these magic numbers into meaningfully named variables?

Comment (MarkFCain) [reply src/SFA.DAS.Aan.SharedUi/Services/MemberProfileHelper.cs:30]: I don't mind if they are only within the method as local variables, I don't think they need to be public constants

File: src/SFA.DAS.ApprenticeAan.Web/Views/MemberProfile/PublicProfile.cshtml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        await Html.RenderPartialAsync("~/Views/MemberProfile/_MemberPersonalInfo.cshtml", Model.MemberInformation);
+
+        await Html.RenderPartialAsync("~/Views/MemberProfile/_AreasOfInterest.cshtml", Model.AreasOfInterest);
+
+        await Html.RenderPartialAsync("~/Views/MemberProfile/_EmployerInformationSection.cshtml", Model.ApprenticeshipInformation);
+        await Html.RenderPartialAsync("~/Views/MemberProfile/_ApprenticeInformationSection.cshtml", Model.ApprenticeshipInformation);
+
+        await Html.RenderPartialAsync("~/Views/MemberProfile/_ConnectWithMember.cshtml", Model);
+        await Html.RenderPartialAsync("~/Views/MemberProfile/_ConnectViaLinkedInSection.cshtml", Model.ConnectViaLinkedIn);
+    }
```
Comment (MarkFCain) [inline src/SFA.DAS.ApprenticeAan.Web/Views/MemberProfile/PublicProfile.cshtml:21]: very nice!

**Review by MarkFCain — DISMISSED** [review]
one suggestion to amend a validation comment, and the rest are optional suggestions

File: src/SFA.DAS.Aan.SharedUi/Models/PublicProfile/ConnectWithMemberSubmitModelValidator.cs (hunk 1 above)
Comment (DevenKShah) [reply src/SFA.DAS.Aan.SharedUi/Models/PublicProfile/ConnectWithMemberSubmitModelValidator.cs:10]: According to the figma the the correct message it `'You must agree to adhering to the code of conduct` I will update

File: src/SFA.DAS.Aan.SharedUi/Services/MemberProfileHelper.cs (hunk 2 above)
Comment (DevenKShah) [reply src/SFA.DAS.Aan.SharedUi/Services/MemberProfileHelper.cs:30]: These are exactly what the variable name says, address profile ids. Also this is going to go away, so leaving it as is for now.

**Review by MarkFCain — DISMISSED** [review]

**Review by ManishAgarwalDfE — APPROVED** [review]

**Pull request merged**